api/build.sbt

```diff
@@ -76,9 +77,11 @@ packageName in Universal := normalizedName.value
 riffRaffPackageType := (packageZipTarball in Universal).value
 mappings in Universal ++= directory("conf")
 // See the README (## Deploying the app) to understand how the *.yaml files are provided at build time.
-riffRaffArtifactResources += (file("platform/cloudformation/discussion-avatar-api.yaml"), "cfn/avatar-api.yaml")
+riffRaffArtifactResources += (file(
+  "platform/cloudformation/discussion-avatar-api.yaml"
+), "cfn/avatar-api.yaml")
 riffRaffArtifactResources += (file("platform/riff-raff.yaml"), "riff-raff.yaml")
 riffRaffArtifactResources += (riffRaffPackageType.value -> s"${name.value}/${name.value}.tgz")
 scalariformPreferences := scalariformPreferences.value
   .setPreference(DanglingCloseParenthesis, Preserve)
-  .setPreference(SpacesAroundMultiImports, false)
\ No newline at end of file
+  .setPreference(SpacesAroundMultiImports, false)
```

Comment:
Not related to your PR but it's a bit strange that this project has scalafmt and scalariform, which are both potentially conflicting code formatters.  We should remove the scalariform config.

Reply:
I've added this as a card in our health backlog: https://trello.com/c/OCqDWRd4/4440-remove-scalariform-from-discussion-avatar-repo